finalProject/fizzBuzz.js

```diff
@@ -0,0 +1,13 @@
+/*
+ For any number that is a multiple of 3, log 'Fizz'
+ For any number that is a multiple of 5, log 'Buzz'
+ For any number that is a multiple of both 3 and 5, log 'FizzBuzz'
+ For all other numbers, log the number.
+*/
+
+
+var runTask = 1; // Initializing variable for loop run.
+ var endTask = 100;
+  for(var i = 1; i <= 100; i++) // for loop and statement condition to check is given task divisible by 3 and 5? if true then logs " Fizz" and "Buzz"
+
+  console.log(( i % 3 ? '': 'Fizz')+( i % 5 ? '': "Buzz") || i); // it it's false then number.
```

Comment:
This is quite advanced syntax- ternary operators are useful to write concise code, but are not always the most readable. I'd imagine research lead you to this solution which is great, but stick with basics when possible for now to build on foundations. It is hard for me to evaluate Mod0 material taught based on this answer.